src/v2/Apps/Components/AppShell.tsx

```diff
@@ -2,12 +2,11 @@ import { Box } from "@artsy/palette"
 import { NetworkOfflineMonitor } from "v2/Artsy/Router/NetworkOfflineMonitor"
 import { findCurrentRoute } from "v2/Artsy/Router/Utils/findCurrentRoute"
 import { useMaybeReloadAfterInquirySignIn } from "v2/Artsy/Router/Utils/useMaybeReloadAfterInquirySignIn"
-import { NAV_BAR_HEIGHT, NavBar } from "v2/Components/NavBar"
+import { NAV_BAR_HEIGHT, NavBar, MOBILE_NAV_HEIGHT } from "v2/Components/NavBar"
```

Comment:
Is the difference in cases here because `NavBar` is a component and the other two values are shared constants?

Reply:
Yep!